spec/javascripts/controllers/provider_foreman/provider_foreman_form_controller_spec.js

```diff
@@ -135,42 +135,56 @@ describe('providerForemanFormController', function() {
     });
 
     it('delegates to miqService.miqAjaxButton', function() {
-      expect(miqService.miqAjaxButton).toHaveBeenCalledWith('/provider_foreman/edit/new?button=save', true);
+      expect(miqService.miqAjaxButton).toHaveBeenCalledWith('/provider_foreman/edit/new?button=save',
+        Object({ provtype: '',
+                 name: '',
+                 url: '',
+                 zone: 'foo_zone',
+                 verify_ssl: '',
+                 log_userid: '',
+                 log_password: '',
+                 log_verify: '' }));
     });
   });
 
   describe('#addClicked', function() {
     beforeEach(function() {
-      $scope.angularForm = {
+      var angularForm = {
         $setPristine: function (value){}
       };
-      $scope.addClicked();
+      vm.addClicked(angularForm);
     });
 
     it('turns the spinner on via the miqService', function() {
       expect(miqService.sparkleOn).toHaveBeenCalled();
     });
 
     it('delegates to miqService.miqAjaxButton', function() {
-      expect(miqService.miqAjaxButton).toHaveBeenCalledWith('/provider_foreman/edit/new?button=save', true);
+      expect(miqService.miqAjaxButton).toHaveBeenCalledWith('/provider_foreman/edit/new?button=save',
+        Object({ provtype: '',
```

Comment:
I simply meant that `Object({foo: bar})` is exactly the same as just `{foo: bar}` by itself...